<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiMainWindow</class>
 <widget class="QMainWindow" name="uiMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="uiCentralWidget">
   <layout class="QVBoxLayout" name="layoutMain">
    <item>
     <widget class="QTextEdit" name="uiXml">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="uiFilter"/>
    </item>
    <item>
     <widget class="QTreeView" name="uiTree">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
